frontend/src/exams/add-exam-modal.vue

```diff
@@ -82,7 +82,7 @@
                  align-h="center"
                  align-content="center">
             <b-col>
-              <p><h5>Success.  Exam Details Added.</h5></p>
+              <p><h5>Success.  Exam Details Added.</h5>
```

Comment:
Some funky HTML going on here.

Reply:
Yea so when I looked at this file pycharm said that p tag needed to be removed. I can re-add it, but it was showing up as an error for some reason?

Reply:
![screen shot 2019-02-27 at 11 23 33 am](https://user-images.githubusercontent.com/29988052/53516862-2c020480-3a82-11e9-8c36-44bc4fd31c3b.png)

Reply:
^^ just for evidence purposes

Reply:
fixed, pushed up